src/site/content/en/media/video-and-source-tags/index.md

```diff
@@ -0,0 +1,336 @@
+---
+layout: post
+title: The <video> and <source> tags
+authors:
+  - samdutton
+  - joemedley
+description: |
+  You've properly prepared a video file for the web. You've given it correct
+  dimensions and the correct resolution. You've encrypted it. You've even
```

Comment:
In the file prep section, we talk about encrypting video content, but we don't talk about playback of encrypted content anywhere.